### ISSUE TITLE
Replace Maven plugin commands with Quarkus CLI

### ISSUE THREAD

**Issue by gastaldi** [quarkusio/quarkus-workshops#602]
Windows environments are not that friendly when using double quotes (`"`) with the `./mvnw` command and I have seen that it also complains about `JAVA_HOME` not being set when using PowerShell. 

Replacing all `./mvnw` commands with the Quarkus CLI respective command would be nice. Besides being cleaner, it generally works in the environment described above.

**Comment by holly-cummins** [comment]
The downside of that is that it's an extra setup step for workshop participants. One goal of the workshop is close-to-zero setup. Sometimes it's taken people half the workshop to install prereqs on their laptop (when we had more). Baffling, but true. 

On the other hand, another goal of the workshop is to work on windows. I wonder if we could refine the commands for windows to get ones that work reliably? We do have the option to have specific commands for windows users, so we can adjust as needed without worrying about windows and mac.

**Comment by gastaldi** [comment]
@holly-cummins True, but in my humble experience during the development of a Quarkus app, I find it way easier to use the Quarkus CLI than to remember the syntax for the Maven plugin. Even if this is an extra configuration step, I believe we should be promoting the CLI usage instead of forcing users to use the plugins directly. 

As an idea, if the step is still optional, perhaps we could display in a tab-like component the command to be run using the Maven plugin with the CLI in a separate tab for users (like me) who prefer the CLI instead. 

Also, the most frustrating experience is to have a `mvnw` command failing with an unexpected output, which is something we can better control when using the CLI

**Comment by holly-cummins** [comment]
Another downside of this is that for people in closed-down environments, they don't have permissions to install the CLI. We've added a configurator toggle now to show gradle options, so we could add the CLI into that matrix, and then we'd get tabs in the default workshop, and people could choose a CLI workshop. But I think the CLI can't be the default since it's an extra setup step (even though I agree it is nicer to use).

**Comment by holly-cummins** [comment]
I just noticed, when going through the instructions, that they do actually talk through installing the quarkus CLI: https://quarkus.io/quarkus-workshops/super-heroes/variants/bt-gradle-os-mac-ai-true-azure-false-cli-true-container-true-contract-testing-false-extension-false-kubernetes-false-messaging-false-native-true-observability-false/spine.html#introduction-installing-cli. Which is a bit odd, when the instructions are mostly for maven. Ah – on investigations, those instructions were added for gradle support. Gradle users can't create apps without the CLI. 

I think using the Quarkus CLI definitely shouldn't be pre-reqed for locked down environments, so I'll add the opposite of this issue (doh!) to #751.

**Comment by holly-cummins** [comment]
See also https://github.com/quarkusio/quarkus/issues/55234 and its path equivalent, https://github.com/quarkusio/quarkus/pull/56300. At the moment the quarkus cli is actually missing functionality required for the workshop, which would make it a pretty terrible default. :)
